### PR TITLE
Update rekor-monitor codeowners

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- Removed users who are no longer active on project and update the team

second part of #110 and https://github.com/sigstore/community/pull/109

### CODE REVIEW

File: github-sync/github-data/users.yaml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 users:
   - username: A0su
```
Comment (cpanato) [inline github-sync/github-data/users.yaml:2]: @haydentherapper @bobcallaway by removing the user from that team, meaning they don't have any other contribution for sigstore? should we keep the user as well or remove it from the org as well?

File: github-sync/github-data/users.yaml [hunk 2]
```diff
@@ -159,9 +160,6 @@ users:
         role: maintainer
   - username: efebarlas
```
Comment (cpanato) [inline github-sync/github-data/users.yaml:161]: same question here
@haydentherapper @bobcallaway by removing the user from that team, meaning they don't have any other contribution for sigstore? should we keep the user as well or remove it from the org as well?

Comment (SantiagoTorres) [reply github-sync/github-data/users.yaml:161]: I think we can remove efe

File: github-sync/github-data/users.yaml (hunk 1 above)
Comment (cpanato) [reply github-sync/github-data/users.yaml:2]: ok, sounds good. I will do that in a follow-up to make more explicit

**Review by haydentherapper — APPROVED** [review]

File: github-sync/github-data/users.yaml (hunk 2 above)
Comment (cpanato) [reply github-sync/github-data/users.yaml:161]: can you approve this pr @SantiagoTorres :)
i will do the removal in a followup

**Review by SantiagoTorres — APPROVED** [review]
Thank you!

**Pull request merged**